api/lib/infrastructure/repositories/admin-member-repository.js

```diff
@@ -13,6 +14,21 @@ module.exports = {
     return members.map((member) => new AdminMember(member));
   },
 
+  get: async function ({ userId }) {
+    const member = await knex
+      .select('pix-admin-roles.id', 'users.id as userId', 'firstName', 'lastName', 'email', 'role')
+      .from('pix-admin-roles')
+      .where({ userId, disabledAt: null })
+      .join('users', 'users.id', 'pix-admin-roles.userId')
+      .first();
+
+    if (!member) {
+      throw new ForbiddenAccess('Accès non autorisé');
```

Comment:
**remarque**
Vu déjà à l'oral mais ce throw de forbiddenAccess ici est étrange. D'habitude on throw des NotFound quand nos `get` ne trouvent pas. J'aurai préféré voir ce ForbiddenAccess dans le usecase 🤔  ou alors renommer le `get` en `getForAccess` ou quelque chose du genre très spécifique (pour éviter que ce get soit utilisé dans un autre contexte que de la gestion d'accès et qu'on se retrouve avec un forbidden sur une page où on ne cherche qu'à faire de l'affichage)

Reply:
Perso je suis plus partante pour le NotFound, le nom `getForAccess` dans un repo admin-member me parait un peu redondant

Reply:
Ce n'est pas vraiment un `getForAccess` car on ne fait que récupérer les détails d'un `AdminMember` et une vérification au niveau du rôle. Pourquoi ne pas garder le `get` ? En soit il suffit plutôt de passer un objet avec les données pour la requête, comme ici on pourrait ajouter dans un ticket suivant l'attribut `disabledAt`.

Ok pour la modification vers le `NotFound`

Reply:
Je pensais avoir un commentaire à ce sujet mais je le retrouve pas.
J'ai mis le throw ici par commodité et mettre en review le jeudi soir sans me relancer dans un refacto mais je vous rejoins ça devrait être géré plus haut, au niveau du use case par exemple.
Mais, car il y a un mais, si vous décidez qu'on ne gère pas le disabledAt dans cette PR, on peut renvoyer un utilisateur not found. Là c'était une condition pour la connexion, le compte User existe mais il est disabled alors on ne permet pas à l'utilisateur d'accéder à pix-admin.
Il y a deux choses dans ce ticket : accéder aux routes api et se logger à admin.

Reply:
Je fais la modification.

Je pense aussi que l'on devrait voir pour flow de connexion à l'application `admin`. Pourquoi ce usecase devrait retourner une 403 ? C'est un usecase uniquement pour récupérer les informations de l'admin. En soit, la connexion à l'application `admin` devrait vérifier l'existence de l'utilisateur, vérifier que c'est le bon mot de passe mais également vérifier qu'il a un role qui est inclus dans la liste des rôles de l'application `admin`.